tests/Nox.Types.Tests/Types/Nuid/NuidTests.cs

```diff
@@ -4,8 +4,16 @@ namespace Nox.Types.Tests.Types;
 public class NuidTests
 {
```

Comment:
A test ensuring that for a known string we have a known expected nuid its important, so no one breaks implementation